### PR TITLE
[WIP] Community Helm Chart

### PR DESCRIPTION
**Pull request by stevefan1999-personal**
### Description
I created a sample Helm chart for people who likes to run HGE on Kubernetes. However, this is currently not releasable yet and is not really in a functional state.

### Changelog

- [ ] `CHANGELOG.md` is updated with user-facing content relevant to this PR. (Blocking: awaiting completion)

### Affected components

- [ ] Server
- [ ] Console
- [ ] CLI
- [x] Docs
- [x] Community Content
- [ ] Build System
- [ ] Tests
- [ ] Other (list it)

### CODE REVIEW

**Comment by marionschleifer** [comment]
@stevefan1999-personal thank you for submitting this PR 🙂 will you let us know when this is releasable? Then we'll review.

**Comment by stevefan1999-personal** [comment]
@marionschleifer Sorry for the long delay. Personally I use this to deploy my development stack, but I want to see if there are more comments, so I wish there would be someone to review it and leave a LGTM before it is can be released.

File: community/helm/charts/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+Helm is a tool for managing Kubernetes charts. Charts are packages of pre-configured Kubernetes resources.
+
+To install Helm, refer to the [Helm install guide](https://github.com/helm/helm#install) and ensure that the `helm` binary is in the `PATH` of your shell.
+
+### Add Repo
+
+The following command allows you to download and install all the charts from this repository:
+
+(Not Implemented)
```
Comment (praveenweb) [inline community/helm/charts/README.md:29]: This needs to be updated.

File: community/helm/charts/incubator/hasura-graphql-engine/templates/NOTES.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
+1. Get the application URL by running these commands:
```
Comment (praveenweb) [inline community/helm/charts/incubator/hasura-graphql-engine/templates/NOTES.txt:1]: Is this file required or was just internal notes?

Comment (stevefan1999-personal) [reply community/helm/charts/incubator/hasura-graphql-engine/templates/NOTES.txt:1]: This is actually...the default contents from `helm create`...

File: community/helm/charts/README.md (hunk 1 above)
Comment (stevefan1999-personal) [reply community/helm/charts/README.md:29]: huh honestly I'm not familiar with the process of how helm add a repository😂
